app/src/main/java/com/globant/tutoria/subscriberssearch/api/SubscribersClient.java

```diff
@@ -0,0 +1,46 @@
+package com.globant.tutoria.subscriberssearch.api;
+
+import com.globant.tutoria.subscriberssearch.api.interfaces.SubscribersInterface;
+
+import okhttp3.OkHttpClient;
+import retrofit2.Retrofit;
+import retrofit2.converter.gson.GsonConverterFactory;
+
+/**
+ * Created by juan.herrera@globant.com on 26/08/2016.
+ */
+public class SubscribersClient {
+
+    private static String API_URL = "https://api.github.com/repos/octokit/octokit.rb/";
+    private static volatile SubscribersClient subscribersClient;
```

Comment:
Why do you need this volatile variable did you find any problem with thread safety?